ui/lib/sync/addon/components/secrets/sync-activation-modal.ts

```diff
@@ -34,9 +35,10 @@ export default class SyncActivationModal extends Component<Args> {
     // clear any previous errors in the parent component
     this.args.onConfirm();
 
-    // must return null instead of root for non managed cluster.
-    // child namespaces are not sent.
-    const namespace = this.args.isHvdManaged ? 'admin' : null;
+    // sync activation is managed by the root/administrative namespace so child namespaces are not sent.
+    // for non-managed clusters the root namespace path is technically an empty string so we pass null
+    // otherwise we pass 'admin' if HVD managed.
+    const namespace = this.flags.hvdManagedNamespaceRoot;
```

Comment:
there was already a getter returning this logic! seemed best to reuse this logic